department-of-veterans-affairs/vets-website
# DT #8200 Find a Forms QA styling

## Description
Touch up on styling found from QA of 1.0 Find A Forms work.

## Testing done
1. Spin up FE local. 
2. Navigate to http://localhost:3001/find-forms/
3. Compare what you see to what is pointed out on [8200](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8200)
3.5 NOTE: Number 5 on that the list from the user story was ruled as not within the VA design system par the conversation on the ticket. 

## Screenshots
![Screen Shot 2020-11-27 at 9 25 57 AM](https://user-images.githubusercontent.com/26075258/100458989-af56ec80-3092-11eb-80b3-5fdfd4123bf1.png)
![Screen Shot 2020-11-27 at 9 26 15 AM](https://user-images.githubusercontent.com/26075258/100458993-b120b000-3092-11eb-8bfe-25084f226032.png)


## Acceptance criteria
- [x] Fix the missed styling from 1.0.

## Definition of done
